amazon/ionbenchmark/benchmark_runner.py

```diff
@@ -0,0 +1,110 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This module
+"""
+
+import tempfile
+import platform
+import time
+import timeit
+
+from amazon.ionbenchmark.benchmark_spec import BenchmarkSpec
+
+_pypy = platform.python_implementation() == 'PyPy'
```

Comment:
I will say again: it's not clear to me that anyone is actually using this with PyPy. For the benchmarking I would comfortable saying "we don't benchmark or profile on pypy"